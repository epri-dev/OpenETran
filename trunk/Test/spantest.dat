time 1 6e-9 5e-6

span 1
cable 1 100.0 3.0e8 0.0e3
end

line 1 2 1 100.0 0 0
end

meter
pairs 1 0
poles all

resistor 100.0
pairs 1 0
poles 1

steepfront 2.54e3 1.82e-6 9.7e-6 0.0 1.0
pairs 1 0
poles 1
