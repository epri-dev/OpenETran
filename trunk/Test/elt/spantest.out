  N   span     dT   Tmax
  1   0.00 6e-009 5e-006
Z-phase
  1.00000e+002
Z-modal
  1.00000e+002
Modal Transformation (Ti)
  1.00000e+000
Meter at pole 1, from 1 to 0, max V   = 2.486309e+005
Meter at pole 2, from 1 to 0, max V   = 2.526278e+005
nr_iter = 0, nr_max = 0
Execution time [sec]: 0
